### PR TITLE
Creates default configs and allows for non-default configs to override.

### PR DESCRIPTION
**Pull request by flosincapite**

### CODE REVIEW

**Review by xmunoz — CHANGES_REQUESTED** [review]
Please update the README

File: cronenbroguelike/game.py [hunk 1]
```diff
@@ -19,8 +20,12 @@
 
 
 def _load_config():
-    with open("game_config.json", "r") as inp:
-        return json.load(inp)
+    with open("game_config.default.json", "r") as inp:
```
Comment (xmunoz) [inline cronenbroguelike/game.py]: Why are you doing this two different ways? Just write a function to load the right config.

**Review by xmunoz — CHANGES_REQUESTED** [review]

File: cronenbroguelike/game.py (hunk 1 above)
Comment (flosincapite) [reply cronenbroguelike/game.py]: I want there to be default settings.

I want the user to be able to override those settings in a config file.

Before you requested that we change up how this was done, we had the defaults enshrined in code; settings could be overridden in the config files. You wanted to read from a single file, so we tried to read only from a single config file (and ignore local changes with some `.gitignore` chicanery), but that didn't work.

So we can either go back to what we were doing before (reading defaults from code and overrides from configs) or do something like this (read both defaults and overrides from a config). Feel free to take over the redesign if you like; I'm getting a bit irritated by having to change it back and forth so much.

Comment (flosincapite) [reply cronenbroguelike/game.py]: Ayyyyy, sorry. I was cranky yesterday. Let's discuss the design of this tonight :heart:

**Comment by xmunoz** [comment]
Tests pls!

**Review by xmunoz — APPROVED** [review]

**Pull request merged**